setup.py

```diff
@@ -61,6 +62,7 @@ def main():
         'author_email': 'victor.stinner@gmail.com',
         'classifiers': CLASSIFIERS,
         'packages': ['bytecode', 'bytecode.tests'],
+        'install_requires': ['enum34'] if sys.version_info < (3, 4) else []
```

Comment:
This is a pain if we want to provide wheel packages. Environment markers would be better, but they require setuptools>=18.5 and pip>=6, which is another pain (not sure about setuptools version).